cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -791,14 +791,24 @@
               "description": "Device to use as target for grub installation. If unspecified, ``grub-probe`` of ``/boot`` will be used to find the device"
             },
             "grub-pc/install_devices_empty": {
-              "type": ["string", "boolean"],
-              "description": "Sets values for ``grub-pc/install_devices_empty``. If unspecified, will be set to ``true`` if ``grub-pc/install_devices`` is empty, otherwise ``false``. Using a non-boolean value for this field is deprecated."
+              "description": "Sets values for ``grub-pc/install_devices_empty``. If unspecified, will be set to ``true`` if ``grub-pc/install_devices`` is empty, otherwise ``false``. Using a non-boolean value for this field is deprecated.",
+              "oneOf": [
+                {
+                  "type": "boolean"
```

Comment:
Added benefit of this ordering is that the docs will now list `(boolean/string)` which lends itself to listing the non-deprecated value first.

Reply:
That was intended. Thanks for noticing it!